packages/realm/RealmJS.podspec

```diff
@@ -43,18 +43,12 @@ Pod::Spec.new do |s|
 
   s.source_files           = 'react-native/ios/RealmReact/*.mm'
   s.public_header_files    = 'react-native/ios/RealmReact/*.h'
-
+  s.resource_bundles       = { 'RealmJS' => ['PrivacyInfo.xcprivacy'] }
+  
   s.frameworks             = uses_frameworks ? ['React'] : []
-
   s.library                = 'c++', 'z', 'compression'
 
   s.pod_target_xcconfig    = {
-                                # Setting up clang
-                                'CLANG_CXX_LANGUAGE_STANDARD' => 'c++17',
-                                'CLANG_CXX_LIBRARY' => 'libc++',
```

Comment:
I'm surprised this deletion really needed 🤔 And this change conflicts with [a change on #6650](https://github.com/realm/realm-js/pull/6650/files#diff-664352e790adfc587db3ac0752373b615c2ca3c69d5c787d6ca1fa68d0fd5fc0L53)